project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my-groups.tpl.php

```diff
@@ -9,7 +9,7 @@
 <div class="row">
   <div class="col-md-12 right">
     <div class="my-groups panel-pane">
-      <h2 class="pane-title"><?php print t('My Groups') ?></h2>
```

Comment:
@DavidHernandez 
on your screenshots I can see "My groups" but here it seems like this is totally different.